packages/hawtio/src/plugins/camel/contextsdash/ContextsDash.tsx

```diff
@@ -0,0 +1,345 @@
+import {
+  Button,
+  Card,
+  CardBody,
+  Dropdown,
+  DropdownItem,
+  KebabToggle,
+  Modal,
+  ModalVariant,
+  Text,
+  Toolbar,
+  ToolbarContent,
+  ToolbarItem,
+} from '@patternfly/react-core'
+import { AsleepIcon, InfoCircleIcon, PlayIcon, Remove2Icon } from '@patternfly/react-icons'
+import { Table, TableBody, TableHeader, TableProps, wrappable } from '@patternfly/react-table'
+import { IResponse } from 'jolokia.js'
+import { AttributeValues } from '@hawtiosrc/plugins/connect/jolokia-service'
+import React, { useEffect, useState, useContext } from 'react'
+import { PluginNodeSelectionContext } from '@hawtiosrc/plugins'
+import { log } from '../globals'
+import { contextsService, ContextDashAttributes } from './contexts-service'
+import { eventService } from '@hawtiosrc/core'
+import { workspace } from '@hawtiosrc/plugins/shared'
+
+export const ContextsDash: React.FunctionComponent = () => {
```

Comment:
what is `ContextsDash`?

Reply:
I'll remove JMX plugin version and rename.